src/lib.rs

```diff
@@ -259,19 +258,17 @@ mod arch {
                         command_builder.args(&c[1..]);
                         print_command(&command_builder, verbose);
 
-                        let trace = match blondie::trace_command(command_builder, false) {
-                            Err(err) => {
-                                eprintln!("{}: {:?}", BLONDIE_ERROR, err);
-                                exit(1);
-                            }
-                            Ok(trace) => trace,
-                        };
+                        let trace = blondie::trace_command(command_builder, false)
+                            .map_err(|err| anyhow!("{}: {:?}", BLONDIE_ERROR, err))?;
```

Comment:
`blondie::Error` doesn't `impl std::error::Error`, so we have to use `map_err + Debug` rather than `context`.